Gems/ScriptCanvas/Code/Include/ScriptCanvas/Grammar/ParsingMetaData.h

```diff
@@ -71,6 +71,14 @@ namespace ScriptCanvas
             void PostParseExecutionTreeBody(AbstractCodeModel& model, ExecutionTreePtr execution) override;
         };
 
+        class UserFunctionNodeCallMetaData
+        {
+        public:
+            AZ_TYPE_INFO(UserFunctionNodeCallMetaData, "{893A827F-9340-4FE1-9829-69E2602F37A1}");
+            AZ_CLASS_ALLOCATOR(UserFunctionNodeCallMetaData, AZ::SystemAllocator, 0);
+
+            bool m_isLocal = false;
+        };
```

Comment:
Just curious, why do you need a typed class that only wraps a bool when you could (seemingly) just use a bool?

Reply:
I'm open to changing it, but its just because it reads as more deliberate/defensive when one gets the value out of the user data. A single bool could have been put there by anyone.